src/tools/genpolicy/rules.rego

```diff
@@ -1319,6 +1319,35 @@ ExecProcessRequest {
     print("ExecProcessRequest 3: true")
 }
 
+UpdateInterfaceRequest {
+    print("UpdateInterfaceRequest 1: input =", input)
+
+    i_interface := input.interface
+    p_allowed_raw_flags := policy_data.request_defaults.UpdateInterfaceRequest.allow_raw_flags
+
+    print("UpdateInterfaceRequest 1: i_interface.raw_flags =", i_interface.raw_flags)
```

Comment:
I would expect the print(input) above to print the raw_flags too, so no need to print again here.

The length of the Policy is significant - smaller Policies work better with the K8s annotation size limitations.

Reply:
I've addressed this in the latest version of the commit; minimized prints to just input and policy at the start. Removed " 1" in the prints.